includes/db/DatabaseOracle.php

```diff
@@ -884,7 +884,7 @@ function tableExists( $table, $fname = __METHOD__ ) {
 	private function fieldInfoMulti( $table, $field ) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce the number of returns of this function 4, down to the maximum allowed 3. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1142)